requirements.txt

```diff
@@ -0,0 +1,7 @@
+lxml
+bleach==1.4.2
+oauthlib==1.0.3
+Mako
```

Comment:
@doctoryes does the version of Mako in here need to match the version we are using in edx-platform? (which btw is mako==1.0.2)

Should this be mako>=1.0,<2.0 ? perhaps  < 1.2?